src/main/java/org/threadly/litesockets/utils/SSLProcessor.java

```diff
@@ -148,7 +158,9 @@ public MergedByteBuffers encrypt(final ByteBuffer buffer) {
     }
     if(gotFinished && finishedHandshake.compareAndSet(false, true)) {
       handshakeFuture.setResult(ssle.getSession());
-      client.write(ByteBuffer.allocate(0));
+      if(tempBuffers.remaining() > 0) {
+        mbb.add(encrypt(ByteBuffer.allocate(0)));
```

Comment:
I assume this and the lower case will not be frequent enough to justify having a static empty ByteBuffer?  If they do, seems like a good thing to make globally available.